### PR TITLE
sauerbraten: update to 2020.12.29

### PR DESCRIPTION
**Pull request by ologantr**
#### Testing the changes
- I tested the changes in this PR: **briefly**

#### Local build testing
- I built this PR locally for my native architecture, (x86_64-glibc)

### CODE REVIEW

**Comment by paper42** [comment]
Please dont close and open PRs for new updates, just force push to the old one next time.

**Comment by ericonr** [comment]
@the-maldridge ok?

**Comment by the-maldridge** [comment]
Looks okay to me.  I think we've also gained enough disk space we can unrestrict.

**Comment by ologantr** [comment]
> Looks okay to me. I think we've also gained enough disk space we can unrestrict.

If this is the case I could force push to my branch a fix to remove restriction.
Am I to amend the single commit or add another that just removes "restricted"?

**Comment by the-maldridge** [comment]
Just make the change and force push, no need to mention it in the commit messsage.

File: srcpkgs/sauerbraten/patches/flags.patch [hunk 1]
```diff
@@ -1,35 +1,20 @@
---- a/src/Makefile	2017-01-12 21:54:08.156276696 -0600
-+++ b/src/Makefile	2017-01-12 21:56:06.889752748 -0600
-@@ -1,4 +1,4 @@
--CXXFLAGS= -O3 -fomit-frame-pointer
-+CXXFLAGS= -O3 -fomit-frame-pointer $(CFLAGS)
+--- a/src/Makefile 	2021-12-31 19:09:40.620827928 +0100
++++ b/src/Makefile	2021-12-31 19:11:40.736780250 +0100
+@@ -1,2 +1,2 @@
+-CXXFLAGS= -O3 -fomit-frame-pointer -ffast-math
++CXXFLAGS= -O3 -fomit-frame-pointer -ffast-math $(CFLAGS)
```
Comment (paper42) [inline srcpkgs/sauerbraten/patches/flags.patch:5]: I am not familiar with this package, but why are we adding -ffast-math here?

Comment (Johnnynator) [reply srcpkgs/sauerbraten/patches/flags.patch:5]: `fast-math` is decided by upstream, we are only appending `$(CFLAGS)`

**Pull request merged**